clojure-mode.el

```diff
@@ -327,7 +327,44 @@ If JUSTIFY is non-nil, justify as well as fill the paragraph."
         (do-auto-fill)))))
 
 
+;;; #_ comments font-locking
```

Comment:
This comment disagrees a bit with actual code (as it affects `comment` as well).